_includes/themes/ploeh/post.html

```diff
@@ -25,7 +25,7 @@ <h1>{{ page.title }} <small>by Mark Seemann</small></h1>
     {% include JB/comments %}
   </div>
   
-  <div class="col-lg-4 col-md-4">
+  <div class="col-lg-4 col-md-4 hidden-print">
```

Comment:
I'm sorry I didn't see this the first time around, but I don't think the print view should hide the publication date. This is important information that a reader might need.

Reply:
I added an additional "Published Date" `div` which will show only when printing. (The current Published Date is inside the sidebar.)